### PR TITLE
Logrhythm - Added NTLM Support

### PR DESCRIPTION
**Pull request by guykeller**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23964

## Description
LogRhythm did not support windows authentication until now,  and such authentication would cause errors.
That problem is now fixed.

## Does it break backward compatibility?
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/LogRhythm/Integrations/integration-LogRhythm.yml [hunk 1]
```diff
@@ -354,21 +382,45 @@ script:
     };
 
     var sendRequest = function(envelope, soapAction, apiUrl) {
+        username = params.Credentials.identifier.split('\\')[1]
```
Comment (ronykoz) [inline Packs/LogRhythm/Integrations/integration-LogRhythm.yml:385]: why do we need the `\` ?

File: Packs/LogRhythm/Integrations/integration-LogRhythm.yml [hunk 2]
```diff
@@ -47,6 +47,16 @@ configuration:
   name: incidentType
   type: 13
   required: false
+- display: Use Windows Authentication
+  name: use_ntlm
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Domain of windows authentication user (used for windows authentication)
```
Comment (ronykoz) [inline Packs/LogRhythm/Integrations/integration-LogRhythm.yml:55]: Shouldn't it be 
```suggestion
- display: Domain of windows authentication server (used for windows authentication)
```

File: Packs/LogRhythm/Integrations/integration-LogRhythm.yml (hunk 1 above)
Comment (guykeller) [reply Packs/LogRhythm/Integrations/integration-LogRhythm.yml:385]: Because that's how we are expected to send it when using NTLM for some reason

**Comment by kirbles19** [comment]
@guykeller Done.

**Pull request merged**